po/glossary/nl.po

```diff
@@ -763,22 +757,20 @@ msgid "tax info"
 msgstr "belastinginfo"
 
 #. "Amost everybody has to declare and probably pay it. See https://en.wikipedia.org/wiki/Income_tax"
-#, fuzzy
-#| msgid "account type: Income"
 msgid "tax type: income tax"
-msgstr "rekening type: Inkomsten"
+msgstr "belasting type: Inkomsten belasting"
 
 #. "Usually only business users have to handle it, see https://en.wikipedia.org/wiki/Sales_tax."
 msgid "tax type: sales tax"
-msgstr ""
+msgstr "belasting type: omzetbelasting"
```

Comment:
The wiki page referred to in the comments has a Dutch translation, which uses "Verkoopsbelasting", a tax type not used in  Belgium or The Netherlands. Why did you choose "omzetbelasting"  ?